CONTRIBUTING.md

```diff
@@ -33,6 +33,88 @@ Thus, before your contribution can be accepted by the project team, contributors
 For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
 
+## Clone and build
+
+Clone the repository:
+
+```
+git clone https://github.com/eclipse-thingweb/node-wot
+```
+
+Go into the repository:
+
+```
+cd node-wot
+```
+
+Install root dependencies (locally installs tools such as [typescript](https://www.npmjs.com/package/typescript)):
+
+```
+npm ci
+```
+
+Use `tsc` to transcompile TS code to JS in the `dist` directory for each package:
+_Note: This step automatically calls `npm run bootstrap`._
+
+```
+npm run build
+```
+
+#### Optional steps
+
+###### Link Packages
+
+Make all packages available on your local machine (as symlinks). You can then use each package in its local version via `npm link <module>` instead of `npm install <module>` (see also https://docs.npmjs.com/cli/link). This is also useful if you want to test the CLI with a local script.
+
+```
+sudo npm run link
+```
+
+(On Windows omit `sudo`)
+
+##### Link Local wot-typescript-definitions
+
+To involve the Scripting API in development, you need to use a locally changed version of the [wot-typescript-definitions](https://www.npmjs.com/package/wot-typescript-definitions).
```

Comment:
I am not sure what "locally changed version" means?

Reply:
I think it refers to the use case of you developing a new version of `wot-typescript-definitions` and what to test them with node-wot. I just copied this section over, but we could say:
```suggestion
When experimenting with new APIs, developers might need to change the [wot-typescript-definitions](https://www.npmjs.com/package/wot-typescript-definitions). If you want to use your own version of wot-types-defitions you can link your local copy using the following steps.
```